Dockerfile

```diff
@@ -98,7 +92,6 @@ RUN chmod 0444 gcloud-service-key.json && \
     mv docker/prod/nginx.conf /etc/nginx/nginx.conf && \
     mv docker/prod/supervisord.conf /etc/supervisor/conf.d/ && \
 
-    rm -rf docker && \
-    rm -rf web/app_dev.php
+    rm -rf docker composer.json composer.lock
```

Comment:
`HttpKernel\Kernel` class uses the composer.json file to find the `project_dir`

Reply:
Best catch ever :+1: Thank you !